### DISCUSSION TITLE
which BT barcode scanner?

### DISCUSSION THREAD

**Discussion by prankousky** [Forceu/barcodebuddy#173] [Q&A]
Hi everybody,

I already own a barcode scanner that is also capable of some sort of wireless transmission. However, it comes with a dongle to connect to the computer and does not pair with my smartphone. I could get an adapter and see if the phone accepts it then, but since `barcodebuddy` supports usage of a BT barcode scanner, I thought I'd just ask what people here use that is natively supported by android.

I mean: buy the scanner, pair with the phone, start the app, and scan things. Without any adapters and dongles, just out of the box.

What do you use, what do you suggest I buy?

Thank you in advance for your help :)

**Comment by generalsnipe** [comment]
I too have the same question. I own a Symbol LS4278 and I can not get it to add QTY using the Screen PHP Page. I tested the scanner with Notepad and it is working.

**Comment by Forceu** [comment]
Personally I use a wireless scanner that is connected through USB and registers as a keyboard. I have heard that Bluetooth scanners are causing issues, but I do not own one, so I cannot answer in that regard.

If you are using the Android app, you could pair the scanner with your phone and then scan through the app on the main menu. Otherwise I guess the best bet would be to pair it to the server that runs barcode buddy and use [grabInput_variant3.py](https://github.com/Forceu/barcodebuddy/blob/master/example/grabInput_variant3.py)

**Comment by prankousky** [comment]
Sorry, I forgot to mention that this is purely about the Android app. My scanner will not pair with the phone, at least my phone does not find it as a bluetooth device (even though I am pretty sure it uses BT, perhaps there is encryption between the scanner and the dongle, or it is invisble to anything but the dongle..?).

We usually unpack groceries in the kitchen (where there is no computer), so I thought using a BT scanner connected to an Android device would be the easiest way to do this. I can try and see if the scanner will work when used in the kitchen (if there is enough range, it should transmit to the office when running barcodebuddy in the browser and connecting the scanner via USB).  I just thought perhaps there is a "go to" BT scanner that users here recommend because it works with all sorts of Android phones... if not, I'll try this approach here.

**Comment by Forceu** [comment]
Unfortunately I do not have any recommendations in that regard. The server where I plugged in the scanner's receiver is around 20 meters from the kitchen and I simply leave the scanner in the kitchen all the time, so far this worked without any problems.

I will leave this open in case someone else has some suggestions

**Comment by Dinth** [comment]
Hi. I am new to Barcode Buddy and i came here to ask - will Barcode Buddy (running on a server) support BT scanner connected to an android phone? Will it be possible to pair the app with the server without a working camera on the phone? (i dont have BB running on the server yet).

But not to do an off topic, ive been using a Netum C750 scanner for the past few months with Grocy and im surprised how good it is. It has replace a much larger, semi-"industrial" and expensive scanner in my setup and its actually better - not only much cheaper, smaller and handy, but also recognizes barcodes at least as good as the expensive model.
